### DISCUSSION TITLE
多开

### DISCUSSION THREAD

**Discussion by liubowei2018** [ttttupup/wxhelper#217] [Q&A] [answered]
现在新安装微信 版本变成 3.9.5.91 很是无语

**Comment by ttttupup** [comment]
官网直接下载

**Reply by ttttupup** [reply, marked as answer]
https://github.com/tom-snow/wechat-windows-versions

**Reply by liubowei2018** [reply]
好的 谢谢大佬

**Reply by liubowei2018** [reply]
我想问下 多开  注入并指定http端口 运行后 
![image](https://github.com/ttttupup/wxhelper/assets/35675191/a948154d-ad08-41d1-b16a-192e0c2c1db7)
然后微信就崩溃没有了

**Reply by ttttupup** [reply]
多开32位的，不是64位的

**Reply by liubowei2018** [reply]
是把注入工具 以32位的重新编译成exe 使用吗

**Reply by ttttupup** [reply]
这个多开只支持以前32位版本的，64位的不支持

**Reply by liubowei2018** [reply]
那现在版本，获取多个微信上的信息 能实现吗

**Reply by hzleii** [reply]
你实现了吗？🫡

**Reply by liubowei2018** [reply]
多开监控微信吗

**Reply by hzleii** [reply]
是的，监控多个微信

**Reply by liubowei2018** [reply]
![image](https://github.com/ttttupup/wxhelper/assets/35675191/55e93998-5ab7-4ca6-b146-ba5c1383ad54)
跟一个微信注入是一样的， 每注入一次修改一下配置文件，就是得看下你哪里注入时获取配置的是微信目录还是自己的项目目录，我这里是获取的项目目录下的config.ini

**Reply by neo2616** [reply]
@liubowei2018 请教是怎么把config.ini 从微信目录变成自己的项目目录

**Comment by neo2616** [comment]
命令行参数P指定端口和 config.ini里的端口是同一个吗

**Reply by liubowei2018** [reply]
是的
